### DISCUSSION TITLE
The nikola-discuss mailing list is under spam bombardment

### DISCUSSION THREAD

**Discussion by ant-222** [getnikola/nikola#3782] [Ideas]
Hello,

The mailing list (`nikola-discuss@googlegroups.com`) is infested with spam. Perhaps it is high time the community abandoned Google Groups and migrated to a saner non-corporate mailinsg-list host with good archiving, moderation, and administration facilities, such as [freelists.org](https://freelists.org/)—but this is just an example.

**Comment by Kwpolska** [comment]
I think we don’t really need a mailing list, we could migrate to GitHub issues and discussions.

**Reply by Kwpolska** [reply]
I removed references to `nikola-discuss` from the repo and the website, and sent an e-mail encouraging migrating to GitHub Discussions.
